BZ1964586 Added notes in Deploying the Cluster Section

Wiharris opened the pull request:
**BZ1964586**
Added note not to exceed the maximum character limit for datastore and cluster name.

**Version(s):**
Versions 4.6 and beyond

**Issue:**
https://bugzilla.redhat.com/show_bug.cgi?id=1964586

**Link to docs preview:**
https://wiharris.github.io/openshift-docs/BZ1964586/installing/installing_vsphere/installing-vsphere-installer-provisioned.html#installation-launching-installer_installing-vsphere-installer-provisioned

Additional information:
**QE:** @duanwei33 



<!--- Next steps after opening your PR:

* Ask for peer review from the OpenShift docs team:
  - For Red Hat associates: Ping @peer-review-squad requesting a review in the #forum-docs-review channel (CoreOS Slack workspace) and provide the following information:
    * A link to the PR.
    * The size of the PR that the GitHub bot assigns (ex: XS, S, M, L, XL).
    * If there is urgency or a deadline for the review.
  - For community authors: Request a review by tagging @openshift/team-documentation in a GitHub comment.

  Slack is the quickest and preferred way to request a review.

* IMPORTANT:
  - All documentation changes must be verified by a QE team associate before merging.
  - Squash to one commit before submitting your PR for peer review.

* For more information about verifying your content, see:
  https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#verification-of-your-content

* For more information about contributing to OpenShift documentation, see:
  https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/contributing.adoc

Additional resources

The OpenShift docs repo adheres to the following style guides:

- OpenShift documentation guidelines (OSDOCS)
  https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc
- Red Hat Supplementary Style Guide (SSG)
  https://redhat-documentation.github.io/supplementary-style-guide/
- Modular Documentation Reference Guide (Mod Docs)
  https://redhat-documentation.github.io/modular-docs/
- IBM Style Guide (ISG)
  https://www.ibm.com/docs/en/ibm-style

You can log in to the ISG by using your @redhat.com id and single sign-on (SSO) credentials. --->

duanwei33 commented:
@Wiharris Sorry I missed the tag ...
From the BZ description, I think we should mention the combined string doesn't exceed 60 characters? 
`And hence we should warn users from choosing long names for datastore and cluster name and recommend that users should not pick datastore and cluster-name in such a manner that their combined string length goes beyond 60 characters. `

Also for  "[] and /" limitation you mentioned, is this for OCP side? I mean vSphere allowed this but Openshift doesn't? If in this case, I think we should not choose such datastore?

Wiharris commented:
> @Wiharris Sorry I missed the tag ... From the BZ description, I think we should mention the combined string doesn't exceed 60 characters? `And hence we should warn users from choosing long names for datastore and cluster name and recommend that users should not pick datastore and cluster-name in such a manner that their combined string length goes beyond 60 characters. `
> 
> Also for "[] and /" limitation you mentioned, is this for OCP side? I mean vSphere allowed this but Openshift doesn't? If in this case, I think we should not choose such datastore?

@duanwei33 I apologize for getting back so late! I checked and it is allowed by vSphere and OCP. I have removed the [] and /" references and updated the PR! :)

Wiharris commented:
Hi @duanwei33 

> @Wiharris as mentioned in previous comments, could you help check: (And sorry miss the notification again!)
> 
> ```
> From the BZ description, I think we should mention the combined string doesn't exceed 60 characters?
> And hence we should warn users from choosing long names for datastore and cluster name and recommend that users should not pick datastore and cluster-name in such a manner that their combined string length goes beyond 60 characters.
> ```

@duanwei33 Done! :) I have updated the PR to include the above statement.

duanwei33 commented:
Thanks for the update.
/lgtm

GroceryBoyJr commented on modules/installation-launching-installer.adoc:
```diff
@@ -303,6 +303,11 @@ ifdef::vsphere,vmc[]
++
+[NOTE]
+====
+DataStore and cluster names cannot exceed 60 characters; therefore, ensure the combined string length does not exceed the 60 character limit when picking DataStore and cluster names.
```
@Wiharris the "s" in Datastore is not capitalized in vSphere's documentation. Example usage: https://docs.vmware.com/en/VMware-vSphere/7.0/com.vmware.vsphere.storage.doc/GUID-3CC7078E-9C30-402C-B2E1-2542BEE67E8F.html

Wiharris replied:
Done! :)

GroceryBoyJr commented on modules/installation-launching-installer.adoc:
```diff
@@ -312,6 +317,11 @@ endif::vsphere,vmc[]
++
+[NOTE]
+====
+DataStore and cluster names cannot exceed 60 characters; therefore, ensure the combined string length does not exceed the 60 character limit when picking DataStore and cluster names. 
```
@Wiharris same as above.

Wiharris replied:
Done! :)

GroceryBoyJr reviewed (commented):
@Wiharris only one tiny little nit, sir.

nalhadef commented on modules/installation-launching-installer.adoc:
```diff
@@ -303,6 +303,11 @@ ifdef::vsphere,vmc[]
++
+[NOTE]
+====
+Datastore and cluster names cannot exceed 60 characters; therefore, ensure the combined string length does not exceed the 60 character limit when picking datastore and cluster names.
```
Is the text "when picking datastore and cluster names" necessary?

Wiharris replied:
Thanks, @nalhadef and I agree! Adding "when picking datastore and cluster names" is not necessary as it is mentioned in the beginning part of the sentence. 

I have incorporated the above suggested changes! :)

nalhadef commented on modules/installation-launching-installer.adoc:
```diff
@@ -312,6 +317,11 @@ endif::vsphere,vmc[]
++
+[NOTE]
+====
+Datastore and cluster names cannot exceed 60 characters; therefore, ensure the combined string length does not exceed the 60 character limit when picking datastore and cluster names.
```
See previous.

Wiharris replied:
I have incorporated the above suggested changes! :)

nalhadef commented:
/lgtm

bmcelvee commented:
/cherrypick enterprise-4.6

bmcelvee commented:
/cherrypick enterprise-4.8

bmcelvee commented:
/cherrypick enterprise-4.11
